### PR TITLE
v10

### PR DESCRIPTION
**Pull request by chimosky**
@quozl @JuiP kindly review.

### CODE REVIEW

File: activity/activity.info [hunk 1]
```diff
@@ -1,8 +1,7 @@
 [Activity]
 name = Flip
-activity_version = 9
-license = GPLv3+;MIT
-exec = sugar-activity3 FlipActivity.FlipActivity
+activity_version = 10
+license = GPLv3
```
Comment (quozl) [inline activity/activity.info]: sprites.py is MIT licensed.

FlipActivity.py is GPLv3+ licensed; note the "any later version".

Comment (chimosky) [reply activity/activity.info]: Thanks for pointing it out, fixed it d379bb72. Happened when I was rebasing and I'd thought I'd made all the needed changes.

**Comment by quozl** [comment]
Puzzled by `exec` added in one commit and removed in another, yet there's no effective change other than to move where it is.

**Comment by chimosky** [comment]
Yeah my bad, I'd looked at 0492c19 and didn't see the change to the `activity.info` file.

**Comment by quozl** [comment]
Now `exec` is deleted.

**Comment by chimosky** [comment]
Yeah, fixed in 73bc90e. Thanks.

**Pull request merged**